build_oblivion_remastered_german_voices.bat

```diff
@@ -1,32 +1,57 @@
 chcp 1252
 :: ATTENTION:
 :: PLEASE UPDATE THE BELOW SECTION WITH YOUR PATHS TO THE GAME FILES AND THE PATH TO THE UNREAL ENGINE 5 BINARIES
-:: ALL FOUR PATHS HAVE TO BE UPDATED IN ORDER FOR THIS SCRIPT TO WORK
-set VOICES_1_BSA="D:\SteamLibrary\steamapps\common\Oblivion\Data\Oblivion - Voices1.bsa"
-set VOICES_2_BSA="D:\SteamLibrary\steamapps\common\Oblivion\Data\Oblivion - Voices2.bsa"
-set SHIVERING_ISLES_BSA="D:\SteamLibrary\steamapps\common\Oblivion\Data\DLCShiveringIsles - Voices.bsa"
-set KNIGHTS_BSA="D:\SteamLibrary\steamapps\common\Oblivion\Data\Knights.bsa"
-set OBRE_PAK="D:\SteamLibrary\steamapps\common\Oblivion Remastered\OblivionRemastered\Content\Paks\OblivionRemastered-Windows.pak"
-set UNREAL_BIN_DIR=D:\EpicGames\UE_5.5\Engine\Binaries\Win64
+:: ALL THREE PATHS HAVE TO BE UPDATED IN ORDER FOR THIS SCRIPT TO WORK
+set DIRECTORY_ORIGINAL=F:\Steam\SteamApps\common\Oblivion\Data
+set DIRECTORY_OBRE=F:\Steam\SteamApps\common\Oblivion Remastered\OblivionRemastered\Content

```

Comment:
Irgendwie scheint das Leerzeichen hier im Pfad Probleme mit dem "Extract the BNKs from the OblivionRemastered-Windows.pak" Schritt zu machen.. jedenfalls wird die originale .pak Datei bei mir nicht mehr nach `\tmp\` entpackt (Die 3 Pfade hatte ich aktualisiert)